utils/wgs_gene_coverage/get_gene_coverage_py.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#SBATCH --job-name=gene_coverage
+#SBATCH --time=24:00:00
+#SBATCH --ntasks-per-node=1
+#SBATCH --mem=4G
+#SBATCH --output=%x-%j.out
+#SBATCH --constraint=CentOS7
+#SBATCH --mail-type=ALL
+
+# Usage: sbatch ~/crg2/utils/wgs_gene_coverage/get_gene_coverage.sh <gene_name> </path/to/crams>
+
+set -e 
+
+GENE=$1
+DIR=$2 # DIR to folder containing cram files to be analyzed 
+SCRIPT_DIR=~/crg2/utils/wgs_gene_coverage 
+# Run Python script to retrieve exon and intron positions 
+python ${SCRIPT_DIR}/get_gene_exon_intron.py ${GENE} ${DIR}
```

Comment:
It's best practice here to first load a specific python module (e.g. `module load python/3.9.13`) and then call python. Otherwise, the script uses whatever python version is in the user's $PATH, which will vary between users; because I have some old bcbio paths in my $PATH, the script actually calls python2, which results in an error when the python script executes since python2 does not support f-strings.

Reply:
In the case where you need to use specific python packages that may not be available in an hpf python module, you could instead load a custom virtual env or conda env.